lib/godmin/engine_wrapper.rb

```diff
@@ -17,7 +17,7 @@ def namespaced?
     def namespaced_path
       @namespaced_path ||= begin
         if namespaced?
-          namespace.name.classify.split("::").map(&:underscore)
+          namespace.name.split("::").map(&:underscore)
```

Comment:
Hm, I think this is copied from the generator thing which in turn is copied from Rails..